frontend/main.js

```diff
@@ -57,12 +58,25 @@ async function startApp () {
   // this is to ensure compatibility between frontend and test/backend.test.js
   sbp('okTurtles.data/set', 'API_URL', window.location.origin)
 
-  // TODO: handle any socket errors!
-  createWebSocket(sbp('okTurtles.data/get', 'API_URL'), {
-    // TODO: verify these are good defaults
-    timeout: 3000,
-    strategy: ['disconnect', 'online', 'timeout']
-  })
+  // URL used to connect to the pubsub server. May include query parameters.
+  let pubsubURL = sbp('okTurtles.data/get', 'API_URL')
+
+  if (process.env.NODE_ENV === 'development') {
+    // This is temporarily used in development mode to help the server improve
+    // its console output until we have a better solution. Do not use for auth.
+    const debugID = Array.from(
+      // $FlowIssue doesn't support crypto yet: https://github.com/facebook/flow/issues/5019
+      crypto.getRandomValues(new Uint8Array(3)),
+      (byte) => byte.toString(16).padStart(2, '0')
+    ).join('')
+    pubsubURL += `?debugID=${debugID}`
```

Comment:
I think these 6 lines can be reduced to 1:

```js
    pubsubURL += `?debugID=${crypto.randomBytes(6).toString('hex')}`
```

Reply:
Awesome!

Reply:
But I was using only three bytes, is it ok? That is still enough for 16 million IDs.

Reply:
@snowteamer sure thing!